src/lib/isBefore.js

```diff
@@ -1,9 +1,12 @@
-import assertString from './util/assertString';
 import toDate from './toDate';
 
-export default function isBefore(str, date = String(new Date())) {
-  assertString(str);
-  const comparison = toDate(date);
+export default function isBefore(str, options) {
```

Comment:
```suggestion
export default function isBefore(date, options) {
```
We can use `date` internally so it's more clear what the code does. The README should still use `str`

Reply:
Fair enough. I prefer this anyway.